modules/hitl/src/backend/db.ts

```diff
@@ -11,6 +11,9 @@ const toPlainObject = object =>
     return v && v.sql ? v.sql : v
   })
 
+export const TABLE_NAME_SESSIONS = 'hitl_sessions'
```

Comment:
good work of faith but we plan on deprecating this hitl module in favor of hitl-next

Reply:
I figured I might as well, it's here if we need it.